pkg/flink/internal/store/store_test.go

```diff
@@ -522,47 +522,153 @@ func (s *StoreTestSuite) TestParseSetStatementError() {
 }
 
 func (s *StoreTestSuite) TestParseUseStatement() {
-	key, value, _ := parseUseStatement("USE CATALOG c;")
-	assert.Equal(s.T(), flinkconfig.KeyCatalog, key)
-	assert.Equal(s.T(), "c", value)
+	catalog, database, _ := parseUseStatement("USE CATALOG c;")
```

Comment:
I like the original API better. We should return a type and name `("CATALOG", "c")` to make things easier if we decide to add more use-able types in the future

Reply:
Me too... However, use can now return both catalog and database, and it couldn't in the previous version. That means, I had to either:

1 - Return two strings
2 - Return an array with two positions
3 - Return an object

Since SQL is really stable and very rarely changes, this function will probably return two values for a long time. My thought was that 2 and 3 would require too much boil code for a function that only returns two values. I'd be ok with all 3 but that's why I personally prefer 1.

Reply:
> use can now return both catalog and database

Ah, didn't realize that :+1: